src/scripts/Build-ModuleDoc.ps1

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Builds a README file in markdown for a specified module. Includes sytnax, description, required modules, parameters and examples.
+
+    .PARAMETER Path
+    Path to a .psm1 module file.
+
+    .PARAMETER Name
+    Name of a PowerShell module. (Must be discoverable by Get-Module -ListAvailable)
+
+    .PARAMETER OutputPath
+    Path to folder for resulting .md file. Defaults to $pwd.
+
+    .PARAMETER OutputFileName
+    Name for the resulting .md file. Defaults to README.md
```

Comment:
Can we hard-code all of these? I don't mind targeting a single module, but we only plan to have one, so we probably don't need that today. The output path should be hard-coded so we don't have to specify it every time we call the script, like we have for other build scripts.

Reply:
The inputs will be provided by a build task. I don't mind hard-coding I'm just used to making everything generic.

Reply:
Is the build task going to also submit a PR to update the docs? All these docs live in the repo in code.

Reply:
We can do that yes. I haven’t gotten that far yet though. Initially I was thinking it be a manual process or vs code task. But if you’re ok with an auto pr I think that’d be better.

Reply:
After looking into hard-coding, I think we should wait until the pipeline is built since things will likely change in terms of pathing. Also the module would need to be "built" first before this can run.